### ISSUE TITLE
Cystic changes in greater tuberosity of the humerus

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11000]
**Preferred term label:**
Cystic changes in greater tuberosity of the humerus

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Rotator cuff tear, Shoulder internal impingement syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
According to this study,https://link.springer.com/article/10.1007/s00256-006-0157-6, the finding of cysts might not be related to rotator cuff tears. I wonder if we could specify more what the actual pathological finding is? Does cystic change imply more than just the presence of a cyst?

**Comment by MickeySegal** [comment]
Our sources:
https://journals.sagepub.com/doi/abs/10.1111/j.1758-5740.2011.00143.x
"Cystic changes in the greater tuberosity were closely associated with rotator cuff tears and age. When humeral cysts are detected on plain radiographs in a patient with a painful shoulder, the index of suspicion for rotator cuff tear should be high, particularly for those patients aged over 60 years."

https://pubmed.ncbi.nlm.nih.gov/31594731/
"the following factors were associated with an [return to sports] outcome: male sex, rotator cuff tear documented intra-operatively, absence of a greater tuberosity cyst on pre-operative imaging studies, and cuff tear debridement"

**Comment by pnrobinson** [comment]
@MickeySegal thanks!
Adding new term: 
- ID: HP:6001371
- label: Cystic changes in greater tuberosity of the humerus
- parents: [Abnormal humerus morphology (HP:0031095)](https://hpo.jax.org/app/browse/term/HP:0031095)
- pmid: PMID:35919060
- def: Cystic lesions of the proximal humerus are commonly encountered on routine shoulder magnetic resonance imaging (MRI). True cysts are defined as fluid-filled structures or lesions with an epithelial lining, whereas pseudocysts are fluid-filled lesions without epithelial lining. Cystic lesion encompasses both true cysts, pseudocysts and other lesions which could mimic the appearance of a cyst on MRI, with increased signal on fluid sensitive sequences.
- comment: Cystic lesions of the humeral head are commonly encountered on routine shoulder magnetic resonance imaging (MRI). Differential diagnoses include degenerative lesions, calcific tendinitis with osseous involvement, perianchor cysts, abscesses and less often, tumours. Degenerative lesions, including subcortical and subchondral cysts, are the most commonly encountered. These may be associated with rotator cuff disease and degenerative joint diseases or considered part of ageing depending on the location of the cystic lesions.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11000
- synonyms:
